src/main/java/com/epam/ta/reportportal/core/item/impl/history/TestItemsHistoryHandlerImpl.java

```diff
@@ -82,25 +96,33 @@ public TestItemsHistoryHandlerImpl(TestItemRepository testItemRepository, Launch
 		List<TestItem> itemsHistory = testItemRepository.loadItemsHistory(itemsForHistory.stream()
 				.map(TestItem::getUniqueId)
 				.collect(Collectors.toList()), launchesHistory.stream().map(Launch::getId).collect(toList()));
+
 		Map<Long, List<TestItem>> groupedByLaunch = itemsHistory.stream().collect(Collectors.groupingBy(TestItem::getLaunchId));
 		return launchesHistory.stream().map(l -> buildHistoryElement(l, groupedByLaunch.get(l.getId()))).collect(toList());
 	}
 
-	public void validateItems(List<TestItem> itemsForHistory, List<Long> ids, Long projectId) {
+	private void validateHistoryDepth(int historyDepth) {
+		Predicate<Integer> greaterThan = t -> t > MIN_HISTORY_DEPTH_BOUND;
+		Predicate<Integer> lessThan = t -> t < MAX_HISTORY_DEPTH_BOUND;
+		String historyDepthMessage =
+				"Items history depth should be greater than '" + MIN_HISTORY_DEPTH_BOUND + "' and lower than '" + MAX_HISTORY_DEPTH_BOUND
```

Comment:
Our Suppliers.formattedSupplier() method can be used here to pass values to the resulted string using '{}'-template instead of concatenation